language/tools/move-cli/src/sandbox/commands/publish.rs

```diff
@@ -120,6 +131,11 @@ pub fn publish(
                         if let Err(err) = res {
                             // TODO (mengxu): explain publish errors in multi-module publishing
                             println!("Invalid multi-module publishing: {}", err);
+                            // find CompiledUnitWithSource by module_id
+                            // if let Some(unit) = module_map.get(module_id.name().as_str()) {
+                            //     // explain publish error
+                            //     explain_publish_error(err, state, unit)?;
+                            // }
```

Comment:
why comment these lines?

Reply:
They are commented out on purpose. 
Will remove the comment and add new test case to solve the problem described https://github.com/move-language/move/issues/438